mvnw

```diff
@@ -0,0 +1,286 @@
+#!/bin/sh
```

Comment:
We all have Maven; in my opinion these "Maven bootstrap" scripts should not be committed to the repository. But I won't make *that* block my approval of a PR.